content/doc/index.html.haml

```diff
@@ -32,6 +32,11 @@ section: doc
 
 %div
   %h3
+    %a{ name: "plugin", title: "Plugin" }
```

Comment:
We do not yet have any information to put here, so I would like this to be removed from this pull request please